packages/shared/src/utils/storage/storage.ts

```diff
@@ -36,51 +45,56 @@ export const isStorageSupported = storage => {
     }
 };
 
-const Store = function (storage) {
+const Store = function (this: { storage: Storage }, storage: Storage) {
     this.storage = storage;
     this.storage.getObject = getObject;
     this.storage.setObject = setObject;
 };
 
 Store.prototype = {
-    get(key) {
+    get(key: string) {
         return this.storage.getItem(key) || undefined;
     },
-    set(key, value) {
+    set(key: string, value: string) {
         if (typeof value !== 'undefined') {
             this.storage.setItem(key, value);
         }
     },
-    getObject(key) {
+    getObject(key: string) {
         return typeof this.storage.getObject === 'function' // Prevent runtime error in IE
             ? this.storage.getObject(key)
             : JSON.parse(this.storage.getItem(key) || '{}');
     },
-    setObject(key, value) {
+    setObject(key: string, value: any) {
```

Comment:
This value can be anything, so I added any type here!

Reply:
```suggestion
    setObject(key: string, value: unknown) {
```
😊